app/scripts/components/pill/pill.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import './pill.scss';
+import {useState} from 'react';
+
+export const Pill = ({pageName ='string'}) => {
+  const [isActive, setIsActive] = useState(false);
+  const [isRemove, setIsRemove] = useState(false);
+
+  const changeActiveStatus = () => {
+    setIsActive(prevState => !prevState);
+  };
+
+  return (
+    !isRemove ? <div onClick={changeActiveStatus} className={isActive ? 'pill active' : 'pill'}>
+      <p>{pageName}</p>
+      {
+        isActive ?
+          <svg viewBox="0 0 12 9" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M3.73333 7.06667L0.933333 4.26667L0 5.2L3.73333 8.93333L11.7333 0.933333L10.8 0L3.73333 7.06667Z"/>
+          </svg>
+          : <svg onClick={() => setIsRemove(true)} viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
It will be better to wrap template in a separate value here. ternary operator is not clearly readable